build-tools/automation/build.groovy

```diff
@@ -40,18 +68,43 @@ timestamps {
         utils = load "${XADir}/build-tools/automation/utils.groovy"
 
         utils.stageWithTimeout('init', 30, 'SECONDS', XADir, true) {    // Typically takes less than a second
+            isCommercial = env.IsCommercial == '1'
```

Comment:
Stylistic and semantic question: `env` is for environment variables, right?  Should we be assuming that environment variables are case sensitive or not?  Is there a convention that should be adhered to?

*Normally* -- in my limited experience -- environment variables are `ALL_CAPS` with `_` as a "word separator", e.g. `JAVA_HOME`, `MONO_PATH`, etc.

Yet here, I see `WORKSPACE` (two lines down), `IsCommercial` (this line), `CommercialDirectory` (next line), `GITHUB_AUTH_TOKEN` (line 35), etc.

Should there be "consistency" here?  Or is this a non-issue?

Reply:
Looking at the [build log](https://jenkins.mono-project.com/job/xamarin-android-pr-pipeline-release/824/), the environment variables are themselves "inconsistent", with multiple PascalCased and multiple `SCREAMING_CAPS` env vars:

```
ghprbPullId=3004
JENKINS_NODE_COOKIE=8aa81940-90c5-4396-8595-ad1dc3941259
ghprbPullLink=https://github.com/xamarin/xamarin-android/pull/3004
BUILD_URL=https://jenkins.mono-project.com/job/xamarin-android-pr-pipeline-release/824
StorageCredentialId=....
GitRepo=xamarin/xamarin-android
...
```

Perhaps a foolish consistency is the hobgoblin of a weak mind, and we need no consistency?

Reply:
Perhaps some method to this madness is that build vars (aka vars that control build behavior) are PascalCased, Jenkins vars are camelCased and shell environment vars are ALL_CAPS. Consequently, I'm tempted to leave the madness as-is.  I _could_ update the build vars, but would be touching many that pre-date this PR and in turn expand the patch footprint.